src/main/java/ca/corefacility/bioinformatics/irida/service/impl/analysis/submission/AnalysisSubmissionServiceImpl.java

```diff
@@ -333,6 +333,29 @@ public AnalysisSubmission updatePriority(AnalysisSubmission submission, Analysis
 		return super.update(submission);
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	@PreAuthorize("hasRole('ROLE_ADMIN')")
+	public AnalysisSubmission updateEmailPipelineResult(AnalysisSubmission submission, boolean emailPipelineResult) {
+		submission.setEmailPipelineResult(emailPipelineResult);
+
+		return super.update(submission);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	@PreAuthorize("hasRole('ROLE_ADMIN')")
+	public AnalysisSubmission updateAnalysisName(AnalysisSubmission submission, String analysisName) {
+		submission.setName(analysisName);
```

Comment:
Maybe do a check here to make sure it is not null or empty

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6